telemetry/record.go

```diff
@@ -41,7 +41,7 @@ var (
 			return &protos.VehicleConnectivity{}
 		},
 	}
-	scientificNotationFloatRegex = regexp.MustCompile("^[+-]?(\\d*\\.\\d+|\\d+\\.\\d*)([eE][+-]?\\d+)$")
+	scientificNotationFloatRegex = regexp.MustCompile(`^[+-]?(\d*\\.\d+|\d+\.\d*)([eE][+-]?\d+)$`)
```

Comment:
Did one `\\` get through the cracks here? We should double check that conversion (but the conversion is a good one :) )

Reply:
fixed. unit tests caught my other mistakes but just not this one